pkg/controllers/updaterun/execution.go

```diff
@@ -467,8 +468,10 @@ func (r *Reconciler) handleStageApprovalTask(
 				return false, fmt.Errorf("%w: %s", errStagedUpdatedAborted, unexpectedErr.Error())
 			}
 			approvalRequestStatus := approvalRequest.GetApprovalRequestStatus()
-			approvalAccepted := condition.IsConditionStatusTrue(meta.FindStatusCondition(approvalRequestStatus.Conditions, string(placementv1beta1.ApprovalRequestConditionApprovalAccepted)), approvalRequest.GetGeneration())
```

Comment:
We don't need to remove the generation check here since it's accessing the approvalRequest's generation